business_objects/embedding.py

```diff
@@ -355,16 +419,30 @@ def create_tensors(
     tensors: List[List[float]],
     with_commit: bool = False,
 ) -> None:
-    tensors = [
-        EmbeddingTensor(
-            project_id=project_id,
-            record_id=record_id,
-            embedding_id=embedding_id,
-            data=tensor,
-        )
-        for record_id, tensor in zip(record_ids, tensors)
-    ]
-    general.add_all(tensors, with_commit)
+    to_add = None
+    if len(record_ids) > 0 and "@" in record_ids[0]:
```

Comment:
what is the meaning of the "@" :D (sorry if I'm adding all the comments regarding readability, but to me it isn't clear what the context is directly)

Reply:
is it meant as something to split the text?

Reply:
added comment & yes